src/groups/bmq/bmqt/bmqt_uri.t.cpp

```diff
@@ -657,28 +657,6 @@ static void test7_testLongUri()
     bmqt::UriParser::shutdown();
 }
 
-static void test8_testUriParserOverflow()
```

Comment:
Why is this removed again?  Does it take too long to run?  It seems like a useful test.

Reply:
43 seconds, and overall this test doesn't test a useful behaviour and instead it tests if we have a bug or not.